Resources/Maps/saltern.yml

```diff
@@ -7381,11 +7357,6 @@ entities:
   - containers:
       cellslot_cell_container: !type:ContainerSlot {}
     type: ContainerContainer
-  - cellInsertSound: !type:SoundPathSpecifier
-      path: /Audio/Items/pistol_magout.ogg
-    cellRemoveSound: !type:SoundPathSpecifier
-      path: /Audio/Items/pistol_magin.ogg
-    type: PowerCellSlot
```

Comment:
other reviewers: disregard this. at some point the default values got written to the map for whatever reason, this just removes them

Reply:
I'll also have to check the packed station armory for conflicts when I get home from work because a newer pr that added in armory access and doors was merged before this one :upside_down_face:

Reply:
reminder to make use of the map merge driver (see: Tools/mapping-merge-driver.sh)

Reply:
nevermind also meds :pill: